tools/variant_simulator/simulate_variation

```diff
@@ -77,9 +78,13 @@ print "getting species ($species)";
 print ", chrom ($chromOnly)" if $chromOnly;
 print ", gene ($gene)", if $gene;
 print ", out: $outFile\n";
+my $dbtype = "core";
+$dbtype = "otherfeatures" if $refSeq_genes;
```

Comment:
Could we use a more condensed syntax for it ?
``` $dbtype = $refSeq_genes ?  "otherfeatures" : "core";```

Reply:
Thanks for the suggestion! I've updated this in the next commit.